_examples/modeset/modeset-double-buffered.go

```diff
@@ -234,6 +235,9 @@ func draw(file *os.File) {
 		off           uint32
 	)
 
+	//	gcpct := debug.SetGCPercent(-1)
+	//	defer debug.SetGCPercent(gcpct)
```

Comment:
yes, thanks!
This was just a test to see if the GC was impacting in the drawing performance. But the answer apparently was 'no'.